Poker.AT/App1aaaaa/App.xaml.cs

```diff
@@ -1,100 +0,0 @@
-﻿using System;
```

Comment:
ohhhhhh... now i get it. you clear it at your last commits. excellent 👍
![image](https://cloud.githubusercontent.com/assets/9889268/26220222/7aef8b16-3c1a-11e7-9f63-a36990179863.png)